visualdl/storage/storage.h

```diff
@@ -15,10 +15,10 @@ limitations under the License. */
 #ifndef VISUALDL_STORAGE_STORAGE_H
 #define VISUALDL_STORAGE_STORAGE_H
 
-#include "visualdl/utils/logging.h"
 #include <algorithm>
 #include <set>
 #include <vector>
+#include "visualdl/utils/logging.h"
```

Comment:
this line should be moved to line22, together with other "".

same with other occasions.